en_US/deploy/cluster/security.md

```diff
@@ -1,58 +1,32 @@
 # Cluster security
 
-Besides [authentication](../../access-control/authn/authn.md) and [authorization](../../access-control/authz/authz.md) mechanisms on the node level, EMQX also leverages the features of port mapping and TLS encryption to ensure the security of client data transmission, message communication between cluster nodes, and enterprise system integrations.
+EMQX provides several security mechanisms to ensure the confidentiality, integrity, and availability of data, including [authentication](../../access-control/authn/authn.md) and [authorization](../../access-control/authz/authz.md) mechanisms on the node level, [a security cookie](#set-node-cookie) to ensure secure communication between nodes in a cluster, [TLS/SSL encryption](#configure-tls-ssl-to-secure-cluster-connections) to provide end-to-end encryption for inter-node traffic, as well as [port mapping rules](#port-mapping) to ensure that the firewall rules are configured correctly. 
 
```

Comment:
To help provisioning firewall rules, we will also touch on EMQX's inter-broker communication port mapping rules.